### DISCUSSION TITLE
How to remove tailwind warn "No utility classes were detected"?

### DISCUSSION THREAD

**Discussion by shuaixr** [saadeghi/daisyui#1182] [Q&A] [answered]
In the official sveltekit example taillwind will warn "No utility classes were detected"
It's kind of annoying, how to remove it?
[![xVNtdx.jpg](https://s1.ax1x.com/2022/09/26/xVNtdx.jpg)](https://imgse.com/i/xVNtdx)

**Comment by saadeghi** [comment, marked as answer]
Because there are no utility classes in the example. There's only one `.btn` class used.  
As soon as you use any Tailwind CSS class, the warning will go.
